cmd2/parsing.py

```diff
@@ -29,46 +29,41 @@ class Statement(str):
     :var raw:               string containing exactly what we input by the user
     :type raw:              str
     :var command:           the command, i.e. the first whitespace delimited word
-    :type command:          str or None
+    :type command:          str
     :var multiline_command: if the command is a multiline command, the name of the
-                            command, otherwise None
-    :type command:          str or None
-    :var args:              the arguments to the command, not including any output
+                            command, otherwise empty
+    :type command:          str
+    :var arg_list:          list of arguments to the command, not including any output
                             redirection or terminators. quoted arguments remain
                             quoted.
-    :type args:             str or None
-    :var: argv:             a list of arguments a la sys.argv. Quotes, if any, are removed
-                            from the elements of the list, and aliases and shortcuts
-                            are expanded
-    :type argv:             list
+    :type arg_list:         list
     :var terminator:        the character which terminated the multiline command, if
                             there was one
-    :type terminator:       str or None
+    :type terminator:       str
     :var suffix:            characters appearing after the terminator but before output
                             redirection, if any
-    :type suffix:           str or None
+    :type suffix:           str
     :var pipe_to:           if output was piped to a shell command, the shell command
                             as a list of tokens
     :type pipe_to:          list
     :var output:            if output was redirected, the redirection token, i.e. '>>'
-    :type output:           str or None
+    :type output:           str
     :var output_to:         if output was redirected, the destination file
-    :type output_to:        str or None
+    :type output_to:        str
 
     """
     def __new__(cls,
                 obj: object,
                 *,
-                raw: str = None,
-                command: str = None,
-                args: str = None,
-                argv: List[str] = None,
-                multiline_command: str = None,
-                terminator: str = None,
-                suffix: str = None,
-                pipe_to: str = None,
-                output: str = None,
-                output_to: str = None
+                raw: str = '',
+                command: str = '',
+                arg_list: List[str] = None,
+                multiline_command: str = '',
+                terminator: str = '',
+                suffix: str = '',
+                pipe_to: List[str] = None,
```

Comment:
@kmvanbrunt Why did the type of ``pipe_to`` change from ``str`` to ``List[str]``?

Reply:
I believe I fixed a typo. pipe_to is a list.